File: Java/Factorial.java
import java.util.Scanner;

class Factorial {
	public static void main(String[] args) {
		int n, fact=1, i;
		Scanner scan = new Scanner(System.in);
	
		System.out.print("Please enter a number: ");
		n = scan.nextInt();
	
	
		for(i=n; i>=1; i--){
			fact = fact * i;
		}
	
		System.out.println(n + "! is " + fact + ".");
	}
}

File: CPP/Factorial.cpp
#include<iostream>
using namespace std;
int main()
{
	int n, fact=1, i;
	
	cout<<"Please enter a number: ";
	cin>>n;
	
	
	for(i=n; i>=1; i--)
	{
		fact = fact * i;
	}
	
	cout<<n<<"! is "<<fact<<endl;
	
	return 0;
}

/*
	printf was replaced cout (benefit)
	
	
*/
